{"uuid":"1f73450c-060f-4912-90d9-3857f0d6d7a3","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["11ed2c88-09d5-419e-8a2c-67c8590ed8ab"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687880617656,"stop":1687880617656},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687880617860,"stop":1687880617861},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687880617948,"stop":1687880617948},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687880617987,"stop":1687880617987},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687880617995,"stop":1687880617996}],"start":1687880617563,"stop":1687880617996}